### DISCUSSION TITLE
DDNS-Go默认使用443端口，如何修改呢？

### DISCUSSION THREAD

**Discussion by jihanc** [jeessy2/ddns-go#1128] [Q&A]
如题，
![图片](https://github.com/jeessy2/ddns-go/assets/78247232/65aa2d77-e66e-4860-b7e7-97df5bde1ef5)

**Comment by jeessy2** [comment]
默认9876
你是不是用了nginx

**Reply by jihanc** [reply]
安装了nginx，但是没有代理ddns-go，ddns-go把nginx的443端口给占用了

**Reply by jihanc** [reply]
ddns-go 我也没有修改端口，使用的是默认端口

**Comment by WaterLemons2k** [comment]
不是监听。

如果正在监听 443 端口，`lsof` 结尾会显示 `(LISTEN)`。示例：

```
COMMAND  PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
nginx 1510 root    7u  IPv4  22764      0t0  TCP *:443 (LISTEN)
```

`ESTABLISHED` 表示建立连接。两个端口正在通信，此图为 `Nginx.lan:40914->104.19.192.176:https`。

**Reply by jihanc** [reply]
感谢你的解答,有没有什么办法改变,或取消这个连接吗?

**Reply by WaterLemons2k** [reply]
> 感谢你的解答,有没有什么办法改变,或取消这个连接吗?

其实这个连接只是在正常调用 DNS 服务商的 API，无需担心。

如果真的需要取消，只能停止使用此程序。因为不调用 API 是无法更新 IP 的。

**Reply by jihanc** [reply]
还是有些影响的，这个服务好像占用了443端口，导致Nginx无法监听443，目前我找到了一个折中的方案，先启动Nginx在启用DDNS-Go，Nginx就可以顺利监听443了
